lib/logging/addon/components/logging/new-edit/template.hbs

```diff
@@ -46,8 +46,8 @@
         <div class="banner-icon p-20"><i class="{{esDeployState.icon}}"></i></div>
         <div class="banner-message p-10">
           {{esDeployState.message}}
-          {{#if (eq esDeploy.state 'success')}}
-            <a href="{{originalModel.embeddedConfig.elasticsearchEndpoint}}">
+          {{#if (eq esDeployState.state 'success')}}
+            <a target="_blank" href="{{originalModel.embeddedConfig.elasticsearchEndpoint}}">
```

Comment:
Always `rel="nofollow noreferrer noopener"` on links to anything outside of the UI.